etc/deploy/gen_pachd_tls.sh

```diff
@@ -12,6 +12,7 @@ else
 fi
 
 eval "set -- $( getopt -l "dns:,ip:,port:" -o "o:" "--" "${0}" "${@:-}" )"
+echo "${@}"
```

Comment:
I think we can probably remove this echo (this was just because I didn't understand how getopt worked. IIRC this script still doesn't work on macs—at some point we should rewrite it in go or some such)